.github/workflows/githubci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arduino-platform: ['metro_m0', 'metro_m4', 'nrf52840', 'cpx_ada', 'cpb']
+        arduino-platform: ['metro_m0', 'metro_m4', 'nrf52840']
```

Comment:
do you want to add all 'standard' boards?

Reply:
I tried add those along with esp32 and esp8266 but those doesn't have `EXTERNAL_FLASH_USE_QSPI`  or `EXTERNAL_FLASH_USE_SPI` which cause most examples failed. I plant to make a separated PR to update all examples to use a default SPI/CS if those are not defined for all normal targets.